external-crates/move/crates/move-compiler/tests/move_2024/unit_test/rand_test_invalid.unit_test.exp

```diff
@@ -0,0 +1,64 @@
+error[E10005]: unable to generate test
+  ┌─ tests/move_2024/unit_test/rand_test_invalid.move:4:7
+  │
+4 │     #[rand_test]
+  │       ^^^^^^^^^ No parameters to generate for random test. A #[rand_test] function must have at least one parameter to generate.
+5 │     fun foo() { }
+  │         --- Error found in this test
+
+error[E10003]: invalid attribute value
+  ┌─ tests/move_2024/unit_test/rand_test_invalid.move:7:21
+  │
+7 │     #[rand_test(b = 1)]
+  │                 ----^
+  │                 │   │
+  │                 │   Unsupported attribute value
+  │                 Assigned in this attribute
+
+error[E10004]: invalid usage of known attribute
+   ┌─ tests/move_2024/unit_test/rand_test_invalid.move:10:7
+   │
+10 │     #[rand_test]
+   │       ^^^^^^^^^ Function annotated as both #[test] and #[rand_test]. You need to declare it as either one or the other
+11 │     #[test]
+   │       ---- Previously annotated here
+12 │     fun qux(_c: u64, _d: bool) { }
+   │         --- Error found in this test
+
+error[E10004]: invalid usage of known attribute
+   ┌─ tests/move_2024/unit_test/rand_test_invalid.move:14:7
+   │
+14 │     #[rand_test]
+   │       ^^^^^^^^^ Function annotated as both #[test] and #[rand_test]. You need to declare it as either one or the other
+15 │     #[test]
+   │       ---- Previously annotated here
+16 │     fun quxz() { }
+   │         ---- Error found in this test
+
+error[E10005]: unable to generate test
+   ┌─ tests/move_2024/unit_test/rand_test_invalid.move:18:7
+   │
+18 │     #[rand_test]
+   │       ^^^^^^^^^ No parameters to generate for random test. A #[rand_test] function must have at least one parameter to generate.
+19 │     #[test_only]
+20 │     fun bar() { }
+   │         --- Error found in this test
+
+error[E10004]: invalid usage of known attribute
+   ┌─ tests/move_2024/unit_test/rand_test_invalid.move:19:7
+   │
+18 │     #[rand_test]
+   │       --------- Previously annotated here
+19 │     #[test_only]
+   │       ^^^^^^^^^ Function annotated as both #[test(...)] and #[test_only]. You need to declare it as either one or the other
```

Comment:
```suggestion
   │       ^^^^^^^^^ Function annotated as both #[test(...)] and #[test_only]. You need to declare it as either one or the other
```
We should probably update this message. 
1) Remove args from `test(...)` if it was only for signers
2) Mark as `random_test` when necessary